File: wrappers/dotnet/indy-sdk-dotnet-test/PaymentTests/ListPaymentAddressesTest.cs
using Hyperledger.Indy.PaymentsApi;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Newtonsoft.Json.Linq;
using System.Threading.Tasks;

namespace Hyperledger.Indy.Test.PairwiseTests
{
    [TestClass]
    public class ListPaymentAddressesTest : PaymentIntegrationTestBase
    {
        [TestMethod]
        public async Task TestListPaymentAddressesWorks()
        {
            var paymentAddressJson = await Payments.ListPaymentAddressesAsync(wallet);
            var paymentAddresses = JArray.Parse(paymentAddressJson);
            Assert.AreEqual(0, paymentAddresses.Count);
        }
    }
}

File: wrappers/android/src/androidTest/java/org/hyperledger/indy/sdk/payment/ListPaymentAddressesTest.java
package org.hyperledger.indy.sdk.payment;

import org.hyperledger.indy.sdk.payments.Payments;
import org.json.JSONArray;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class ListPaymentAddressesTest extends PaymentIntegrationTest {

	@Test
	public void testListPaymentAddressesWorks() throws Exception {
		String paymentAddressJson = Payments.listPaymentAddresses(wallet).get();
		JSONArray paymentAddresses = new JSONArray(paymentAddressJson);
		assertEquals(0, paymentAddresses.length());
	}
}
